joss_paper/paper.md

```diff
@@ -67,54 +83,68 @@ so a solution which could prevent this
-1/5 to 1/10 the speed it can iterate through the same trajectory on disk in XTC format \autoref{fig:benchmark}.
-However, it should be noted that this speed is influenced by network latency and that
-writing parallelized algorithms can offset this loss of speed.
+1/5 to 1/10 the speed it can iterate through the same trajectory on disk in XTC format (\autoref{fig:benchmark}).
+However, it should be noted that this speed is influenced by network bandwidth and that
+writing parallelized algorithms can offset this loss of speed as in \autoref{fig:RMSD}. 
 
-![Benchmarks performed on a machine with 2 Intel Xeon 2.00GHz CPUs, 32GB of RAM, and an SSD configured with RAID 0.\label{fig:benchmark}](benchmark.png)
+![Benchmarks performed on a machine with 2 Intel Xeon 2.00GHz CPUs, 32GB of RAM, and an SSD configured with RAID 0. The trajectory used for benchmarking was the YiiP trajectory from MDAnalysisData [@YiiP:2019], a 9000-frame (90ns), 111,815 particle simulation of a membrane-protein system. The original 3.47GB XTC trajectory was converted into an uncompressed 11.3GB H5MD trajectory and an uncompressed 11.3GB ZarrMD trajectory using the MDAnalysis `H5MDWriter` and *Zarrtraj* `ZarrMD` writers, respectively. XTC trajectory read using the MDAnalysis `XTCReader` for comparison. \label{fig:benchmark}](benchmark.png)
 
-With `Zarrtraj`, we envision research groups making their data publicly available 
-via a cloud URL so that anyone can reuse their trajectories and reproduce their results.
-Large databases, like MDDB and MDverse, can expose a URL associated with each 
-trajectory in their databases so that users can make a query and immediately use the resulting
-trajectories to run an analysis on the hits that match their search. Groups seeking to 
-collect a large volume of trajectory data to train machine learning models can make use
-of our tool to efficiently and inexpensively obtain the data they need from these published 
-URLs.
+![RMSD benchmarks performed on the same machine as \autoref{fig:benchmark}. YiiP trajectory aligned to first frame as reference using `MDAnalysis.analysis.align.AlignTraj` and converted to compressed, quantized H5MD (7.8GB) and ZarrMD (4.9GB) trajectories. RMSD performed using development branch of MDAnalysis (2.8.0dev) with "serial" and "dask" backends. See [this notebook](https://github.com/Becksteinlab/zarrtraj/blob/d4ab7710ec63813750d7224fe09bf5843e513570/joss_paper/figure_2.ipynb) for full benchmark codes. \label{fig:RMSD}](RMSD.png)
 
-This work builds on the existing `MDAnalysis` `H5MDReader`
-[@H5MDReader:2021], and similarly uses `NumPy` [@NumPy:2020] as a common interface in-between `MDAnalysis`
-and the file storage medium. `Zarrtraj` was inspired and made possible by similar efforts in the 
-geosciences community to align data practices with FAIR principles [@PANGEO:2022].
+*Zarrtraj* is capable of making use of *Zarr*'s powerful compression and quantization when writing ZarrMD trajectories. 
```

Comment:
Provide link to the trajectory!

Reply:
Like with a free S3 account?

Reply:
Yep, or you can make it requester pays.

Reply:
Should only cost a few $ a year, ask @orbeckst maybe?

Reply:
Make requester pay... it's not straightforward to have funds at hand for running costs.

For a start, I cannot even see where my aws charges originate as management is just too complicated and takes too much time.

Reply:
Even with requester paying for GET requests, there is still a storage cost which applies after the free tier year for S3.

We could use Google Cloud's free tier, which would give us free storage for a single trajectory (5GB) for a limited number of GET requests per month "indefinitely". I believe we can set a quota to prevent any charges actually being accrued, too. Thoughts?

https://cloud.google.com/storage/pricing#cloud-storage-always-free

Reply:
@ljwoods2 if we can make that work I think that would be a great option, just to demonstrate utility easily. We can combine possibly with the remote PDB hack you came up with so that people can just give this a try themselves straight from the example? Thoughts @orbeckst ?

Reply:
I am in favor of a solution that can be left unattended without the potential of generating charges. 

The alternative is a script that dumps a trajectory on a user's AWS storage for testing. Then write a mini tutorial on how to upload a trajectory and then read it.

Reply:
Okay, I created a free tier bucket with my google account and configured it to notify me if usage gets anywhere near what will cause me to be charged. I made sure not to include any promises in the paper that this will be free so I can later switch this to requester pays if there is lots of requests. Also avoided including the exact instructions for streaming and instead linked the docs so that we can change things without the paper being outdated.

Reply:
I included the whole thing for now but added a minor caveat. 

Otherwise it looked incomplete (and I don't think it's good to rely on links for content in a paper).